code/modules/projectiles/projectile/magic/spellcard.dm

```diff
@@ -2,5 +2,5 @@
 	name = "enchanted card"
 	desc = "A piece of paper enchanted to give it extreme durability and stiffness, along with a very hot burn to anyone unfortunate enough to get hit by a charged one."
 	icon_state = "spellcard"
-	damage_type = BURN
+	damage_type = BRUTE

```

Comment:
You switched the damage type but the desc still says it burns.

Reply:
I thought I fixed that description. I'll do that now.

Reply:
Ok so, turns out, I fixed the spell description, but not the projectile description. It's good now.